lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,58 @@
+const {SERVER_JOBS_ADMIN} = require('../userWhitelist.json');
+/**
+ * @param {string} filename
+ */
+const isJobFile = (filename) => {
+  const REGISTRY_FILENAME = 'jobs_registry.py';
+  const JOB_FILE_FIX = '_jobs_';
+
+  if (filename.endsWith(REGISTRY_FILENAME)) {
+    return true;
+  }
+
+  if (filename.includes(JOB_FILE_FIX)) {
+    return true;
+  }
+
+  return false;
+};
+
+/**
+ * @param {import('probot').Context} context
+ */
+const checkForNewJob = async (context) => {
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse} pullRequest
+   */
+  const pullRequest = context.payload.pull_request;
+
+  const pullRequestParams = context.repo({
+    pull_number: pullRequest.number,
+  });
+
+  const filesResponse = await context.github.pulls.listFiles(pullRequestParams);
+  const changedFiles = filesResponse.data;
+  const containsJobFile = changedFiles.some((file) => {
+    return isJobFile(file.filename);
+  });
+
+  if (containsJobFile) {
+    const prAuthor = pullRequest.user.login;
+    const serverJobsForm = 'server jobs form'.link(
+      'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+    const commentBody = 'Hi @' + SERVER_JOBS_ADMIN + ', PTAL at this PR, ' +
+      'it adds/modifies a server job. Also @' + prAuthor + ', please ' +
+      'endeavour to fill the ' + serverJobsForm + ' for the new job ' +
```

Comment:
endeavour to fill the -> fill the.

Also, provide a link to the wiki page with instructions for new jobs: https://github.com/oppia/oppia/wiki/Running-jobs-in-production#submitting-a-pr-with-a-new-job.

Reply:
Also, can we mark such PRs as unmergeable until approved by Server jobs admin?

\cc @seanlip @nithusha21 @DubeySandeep

Reply:
I think what we should do here is to assign the pr to the server jobs admin.

Reply:
That makes sense. Let us do that.